salt/master.sls

```diff
@@ -2,6 +2,27 @@
 {%- from tplroot ~ "/map.jinja" import salt_settings with context %}
 {%- from tplroot ~ "/libtofs.jinja" import files_switch with context %}
 
+{%- if grains.os == 'MacOS' %}
+salt-master-macos:
+  file.managed:
+    - name: /Library/LaunchDaemons/com.saltstack.salt.master.plist
+    - source: https://raw.githubusercontent.com/saltstack/salt/master/pkg/darwin/com.saltstack.salt.master.plist
```

Comment:
you can use https://raw.githubusercontent.com/saltstack/salt/master/pkg/osx/scripts/com.saltstack.salt.master.plist as that is where we build the macOS pkg from.